mgit/lib/utils.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const execSh = require( 'exec-sh' );
+
+const utils = {
```

Comment:
To follow the convention begun in https://github.com/ckeditor/ckeditor5-dev/tree/master/packages/ckeditor5-dev-tests/lib/utils and used in CKEditor 5 code, let's keep every util file separately.